sdk/resourcemanager/Azure.ResourceManager/src/Azure.ResourceManager.csproj

```diff
@@ -9,6 +9,7 @@
     <PackageTags>azure;management;resource</PackageTags>
     <DisableEnhancedAnalysis>true</DisableEnhancedAnalysis>
     <UseDefaultNamespaceAndOutputFolder>false</UseDefaultNamespaceAndOutputFolder>
+    <NoWarn>$(NoWarn);AZC0014</NoWarn>
```

Comment:
we should not add this global nowarn here. We should add it to wherever is reporting this issue, the signature of those methods.

Reply:
I'll get analyzer fix in first. So we don't need to worry about this.